test/integration/monitors_sb2_to_sb3.js

```diff
@@ -140,7 +140,6 @@ test('saving and loading sb2 project with monitors preserves sliderMin and slide
         t.equal(monitorRecord.targetId, null);
 
         t.end();
-        process.nextTick(process.exit);
```

Comment:
Does this one not need a `vm.quit()`?

Reply:
Nope! It never actually starts the VM. I should probably correct the comment below.